CosmicReco/src/SimpleTimeCluster_module.cc

```diff
@@ -41,7 +41,7 @@ class SimpleTimeCluster : public art::EDProducer {
     using Name = fhicl::Name;
     using Comment = fhicl::Comment;
     fhicl::Atom<int> debug{Name("debugLevel"), Comment("set to 1 for debug prints"), 0};
-    fhicl::Atom<int> minnsh{Name("minNStrawHits"), Comment("minimum number of straw hits "), 5};
+    fhicl::Atom<int> minnsh{Name("minNStrawHits"), Comment("minimum number of straw hits "), 2};
```

Comment:
code defaults of algorithm parameters is deprecated.  Please remove this default and add the parameter to the prolog.  While you're in there, please do the same fo the other parameters.